resources/metadata/kaniko.yaml

```
@@ -0,0 +1,85 @@
+metadata:
+  name: kanikoExecute
+  description: Executes a [Kaniko](https://github.com/GoogleContainerTools/kaniko) build for creating a Docker container.
+  longDescription: Executes a [Kaniko](https://github.com/GoogleContainerTools/kaniko) build for creating a Docker container.
+spec:
+  inputs:
+    secrets:
+    - name: dockerConfigJsonCredentialsId
+      type: jenkins
+    params:
+    - name: buildOptions
+      type: '[]string'
+      description: Defines a list of build options for the [kaniko](https://github.com/GoogleContainerTools/kaniko) build.
+      scope:
+        - PARAMETERS
+        - STAGES
+        - STEPS
+      default: --skip-tls-verify-pull
```

Shouldn't this be a linst?

```suggestion
      default: 
        - --skip-tls-verify-pull
```

---

good point, it is now corrected